openfermioncirq/variational/hamiltonian_objective.py

```diff
@@ -38,35 +35,47 @@ class HamiltonianVariationalStudy(VariationalStudy):
             samples.
 
     Attributes:
-        hamiltonian: The Hamiltonian associated with the ansatz, represented
+        hamiltonian: The Hamiltonian of interest, represented
             as a FermionOperator, QubitOperator, InteractionOperator, or
             DiagonalCoulombHamiltonian. If the input is not a QubitOperator,
             then the Jordan-Wigner transform is used.
+        variance_bound: The sum of the absolute values of the coefficients of
```

Comment:
Let's talk about this briefly tomorrow. Note that if you are not using an estimator of the expectation values in your bound, then this variance bound is simply the squared 1-norm of the Hamiltonian.

Reply:
Yes, it is the squared one-norm. I had the doc wrong but I fixed it.